tests/modules/nf-core/basicpy/nextflow.config

```diff
@@ -0,0 +1,7 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+}
+
+docker.runOptions = '--entrypoint ""'
```

Comment:
Do you need this option to run the module? or is it only for tests?

Reply:
Due to the Entrypoint of the container applying directly the python script, this was the only way I could make it work, so I would. think this is required to run the docker version.

Reply:
Maybe this should also be mentioned in the meta.yml then :)

Reply:
Where would I add this to the meta.yml though? There isn't really any space for comments is there? I tried adding one to the tool and it caused failing of the lint. Feel free to add to finalize this :) !

Reply:
Maybe you can add this to the main description of the tool, it's not ideal but I don't think there is a better place available atm